### PR TITLE
Feature: Add CLI Commands & More Examples

### PR DESCRIPTION
**Pull request by f3rno**
Needs:
 * [ ] https://github.com/bitfinexcom/bitfinex-api-node/pull/298
 * [x] https://github.com/bitfinexcom/bitfinex-api-node/pull/225
 * [x] https://github.com/bitfinexcom/bitfinex-api-node/pull/226
 * [x] rebase

New examples have been added, and examples now double as CLI commands via `npm run` :)
The following commands are available:
 * `npm run tickers` - fetch the latest ticker data for all symbols
 * `npm run symbols` - fetch the latest symbol data
 * `npm run order-history <ETHUSD>` - query your order history for a symbol, start/end/limit currently hardcoded
 * `npm run trade-history <ETHUSD>` - queries trades, see `order-history`
 * `npm run positions` - view open positions
 * `npm run positions pl` - as above, but with P/L information
 * `npm run balances` - view wallet balances
 * `npm run balances <USD>` - as above, but with estimated value
 * `npm run orders` - view open orders
 * `npm run cancel-orders` - cancel all open orders (via ws2)
 * `npm run status` - view platform status

Before the commands can be used, a `.env` file in the root directory needs to be made with:
```
API_KEY=...
API_SECRET=...

# Optional
WS_URL=...
REST_URL=...
```

Example output w/ mock data:
screenie-bfx-cli-order-history

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +61,8 @@
     "lodash.throttle": "^4.1.1",
     "request": "^2.67.0",
     "request-promise": "^4.2.0",
+    "socks-proxy-agent": "^3.0.1",
+    "thunkify": "^2.1.2",
```
Comment (robertkowalski) [inline package.json]: - `thunkify ` can be removed
- `socks-proxy-agent` is already a dependency in `devDependencies` at line 41 - one of them has to get removed

File: examples/rest2/tickers.js [hunk 2]
```diff
@@ -2,23 +2,35 @@
...
+
+  return rest.tickers([symbols.map(s => `t${s.toUpperCase()}`)])
+}).then(tickers => {
+  let t
+  for (let i = 0; i < tickers.length; i += 1) {
+    t = tickers[i]
+    table.push([
+      t.symbol, t.lastPrice, t.high, t.low, t.dailyChange, t.bid, t.ask,
+      t.bidSize, t.askSize
+    ])
```
Comment (robertkowalski) [inline examples/rest2/tickers.js]: order of elements is wrong (see symbol / ask)

screen shot 2018-04-27 at 12 07 31

File: examples/rest2/trade-history.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+// TODO: natural lang start/end query args (a library must exist)
+const START = Date.now() - (30 * 24 * 60 * 60 * 1000 * 1000)
+const END = Date.now()
+const LIMIT = 25
+
+const table = new Table({
+  colWidths: [12, 13, 20, 20, 14, 14, 14, 20],
+  head: [
+    'Trade ID', 'Order ID', 'Created', 'Type', 'Exec Amount', 'Exec Price',
+    'Price', 'Fee'
```
Comment (robertkowalski) [inline examples/rest2/trade-history.js]: some fields are empty for me...

screen shot 2018-04-27 at 12 12 15

Comment (f3rno) [reply examples/rest2/trade-history.js]: It needs https://github.com/bitfinexcom/bitfinex-api-node/pull/298

File: examples/rest2/tickers.js (hunk 2 above)
Comment (f3rno) [reply examples/rest2/tickers.js]: #298 as well

File: examples/rest2/trade-history.js (hunk 3 above)
Comment (f3rno) [reply examples/rest2/trade-history.js]: This is an API data issue, if the fields themselves are null in the ws2 packet they will show up empty.

**Pull request merged**